easybuild/easyblocks/c/clang.py

```diff
@@ -150,6 +152,12 @@ def extract_step(self):
 
         src_dirs[clang_src_dir] = os.path.join(self.llvm_src_dir, 'tools', 'clang')
 
+        if LooseVersion(self.version) >= LooseVersion('3.8'):
+            openmp_src_dirs = glob.glob('openmp-*')
+            if len(openmp_src_dirs) != 1:
+                raise EasyBuildError("Failed to find exactly one openmp source directory: %s", openmp_src_dirs)
+            src_dirs[openmp_src_dirs[0]] = os.path.join(self.llvm_src_dir, 'projects', 'openmp')
```

Comment:
I see 4 occurrences. How is that not enough?

It's not about reducing LoC, I don't really care about that, it's about not copy-pasting code.